### PR TITLE
Add support for more features in helm chart

### PR DESCRIPTION
**Pull request by antvick**
* add init container support
* add sidecar container support
* add emptyDir volume support

### CODE REVIEW

**Comment by antvick** [comment]
ready for review

File: charts/kminion/templates/deployment.yaml [hunk 1]
```diff
@@ -24,7 +27,7 @@ spec:
       {{- end}}
       labels:
         {{- include "kminion.selectorLabels" . | nindent 8}}
-        {{- if .Values.customLabels}}
+        {{- if .Values.customLabels -}}
```
Comment (weeco) [inline charts/kminion/templates/deployment.yaml:30]: I think that's not needed

**Review by weeco — COMMENTED** [review]
Hey @antvick ,
thanks for filing this pull request. Generally LGTM, I think you added one dash for the customlabels that shouldn't go there if I'm not wrong.

File: charts/kminion/templates/deployment.yaml (hunk 1 above)
Comment (antvick) [reply charts/kminion/templates/deployment.yaml:30]: this removes an extra empty line when the chart is rendered

Comment (antvick) [reply charts/kminion/templates/deployment.yaml:30]: but happy to remove it

Comment (antvick) [reply charts/kminion/templates/deployment.yaml]: I have remove it

**Comment by antvick** [comment]
@weeco all sorted

**Review by weeco — APPROVED** [review]

**Pull request merged**